standards/javascript.md

```diff
@@ -285,7 +284,7 @@ instead of [function expressions][18]. Because [hoisting][19].
 ##### Bad
 
 ```js
-var sum = function( x, y ) {
+const sum = ( x, y ) => {
```

Comment:
Don't we encourage arrow functions in other scenarios? If so, I don't want people to misinterpret this as all arrow functions are bad.

Reply:
Just ones that will be hoisted like it says in the content, has nothing to do with it being an arrow or not an arrow.

Reply:
Right, I just worry that dummies like me would see the arrow here, but not in the bad example, and think that the arrow was part of what a "function expression is" and assume they're always bad.